datahub-graphql-core/src/main/java/com/linkedin/datahub/graphql/resolvers/knowledge/CreateDocumentResolver.java

```diff
@@ -93,6 +93,12 @@ public CompletableFuture<String> get(DataFetchingEnvironment environment) throws
                     ? java.util.Collections.singletonList(input.getSubType())
                     : null;
 
+            // Extract settings (defaults to true if not provided)
+            final Boolean showInGlobalContext =
+                input.getSettings() != null && input.getSettings().getShowInGlobalContext() != null
+                    ? input.getSettings().getShowInGlobalContext()
+                    : true;
```

Comment:
When creating context from MCP server, we'll hide from the global context - e.g. it won't be visible on the sidebar.